### PR TITLE
Add sorted activities screen with empty state

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**

### CODE REVIEW

File: app/views/sorted-activities.mustache [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <li class="active">
+          <span class="option">{{#__}}tabs.sorted{{/__}}</span>
+        </li>
+      </ul>
+    </nav>
+  </div>
+</div>
+
+<h3 class="heading-medium">{{#__}}sorted-activities.activities-heading{{/__}}</h3>
+<p class="text">{{#__}}sorted-activities.activities-text-1{{/__}}</p>
```
Comment (nathancashmoredwp) [inline app/views/sorted-activities.mustache]: think class="text" on p is unnecessary

File: app/views/sorted-activities.mustache [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+<p class="text">{{#__}}sorted-activities.activities-text-2{{/__}}</p>
+<p class="text">{{#__}}sorted-activities.activities-text-3{{/__}}</p>
+
+<div class="todo-list">
+  <form action="details" method="get">
+
+    <h3 class="heading-small category-heading" data-test="category">{{#__}}category.DOING.title{{/__}}</h3>
```
Comment (nathancashmoredwp) [inline app/views/sorted-activities.mustache]: category-heading ?

File: app/views/sorted-activities.mustache (hunk 1 above)
Comment (lukasz-kaniowski) [inline app/views/sorted-activities.mustache]: `text` is wrapping content at certain width. Will leave it for now.

**Comment by nathancashmoredwp** [comment]
👍

**Pull request merged**